src/Client/HttpExceptionHandler.php

```diff
@@ -73,7 +73,10 @@ public function transformResponseToException(RequestInterface $request, Response
      */
     protected function getResponseMessage(ResponseInterface $response)
     {
-        $decodedBody = json_decode($response->getBody()->getContents(), true);
+        $responseBody = $response->getBody();
+        $decodedBody = json_decode($responseBody->getContents(), true);
+
+        $responseBody->rewind();
```

Comment:
The response body is a Stream. When it's read by calling getContents, the internal cursor stay at the end of the stream. So then getContents is called a second time it returns an empty string. the function rewind put the internal cursor at the beginning of the stream.